mixer/cmd/build.go

```diff
@@ -508,23 +537,45 @@ func setUpdateFlags(cmd *cobra.Command) {
 	_ = cmd.Flags().MarkDeprecated("keep-chroots", "this flag is ignored by the update builder")
 }
 
-var buildCmds = []*cobra.Command{
+var containerCmds = []*cobra.Command{
 	buildBundlesCmd,
 	buildUpdateCmd,
 	buildAllCmd,
-	buildImageCmd,
 	buildDeltaPacksCmd,
-	buildUpstreamFormatCmd,
+}
+
+var nativeCmds = []*cobra.Command{
 	buildFormatBumpCmd,
+	buildUpstreamFormatCmd,
+	buildImageCmd,
+}
+
+var bumpCmds = []*cobra.Command{
+	buildFormatNewCmd,
+	buildFormatOldCmd,
 }
 
 func init() {
-	for _, cmd := range buildCmds {
+	for _, cmd := range containerCmds {
+		if cmd.Annotations == nil {
+			cmd.Annotations = map[string]string{}
+		}
+		cmd.Annotations[containerMarker] = "true"
```

Comment:
I think for your use-case here you can just set it to the empty string and check for key existence instead. I don't like default strings like this.